app/views/_masthead.html.erb

```diff
@@ -1,4 +1,4 @@
-<%= render '/shared/select_work_type_modal' if create_work_presenter.many? %>
```

Comment:
`create_work_presenter` stopped responding to `#many?` on some view in my branch.  I figured our application would always have many, so it was safe to remove.